### PR TITLE
fix: Auto detect SVG content-type

### PR DESCRIPTION
**Pull request by disq**

### CODE REVIEW

File: cli/internal/publish/pluginui_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+func TestQuickContentType(t *testing.T) {
+	tests := []struct {
+		filename string
+		data     []byte
+		want     string
+	}{
+		{
+			filename: "octetstream.svg",
+			data:     []byte(`<svg xmlns="http://www.w3.org/2000/svg" width="2500" height="330" viewBox="0 0 719.713 94.943"><g fill-rule="evenodd" clip-rule="evenodd"><path d="M17.32,16.76l-.53.53-1.09-1.1a4.33,4.33,0,0,0,.53-.52Z" style="fill:#4285f4"/></g></svg>`),
```
Comment (erezrokah) [inline cli/internal/publish/pluginui_test.go:19]: Can we embed these files instead to make it easier to read the tests?

File: cli/internal/publish/pluginui.go [hunk 2]
```diff
@@ -120,14 +120,22 @@ func quickContentType(filename string) (string, error) {
 	}
 
 	contentType := http.DetectContentType(filebytes)
-	if contentType == "application/octet-stream" {
+	switch {
+	case contentType == "application/octet-stream":
```
Comment (erezrokah) [inline cli/internal/publish/pluginui.go:124]: We can use the `mime` package similar to https://github.com/golang/go/issues/15888#issuecomment-222457018 WDYT?

Comment (disq) [reply cli/internal/publish/pluginui.go]: Done in https://github.com/cloudquery/cloudquery/pull/19342/commits/a5579c5b3cfd96a495f3756dc62b78112ab7ffeb

File: cli/internal/publish/pluginui_test.go (hunk 1 above)
Comment (disq) [reply cli/internal/publish/pluginui_test.go]: Done in https://github.com/cloudquery/cloudquery/pull/19342/commits/a5579c5b3cfd96a495f3756dc62b78112ab7ffeb

**Comment by hermanschaaf** [comment]
Linting error in the latest commit:

```
 Error: cli/internal/publish/pluginui_test.go:55:64: copyTempFile - result 1 (error) is always nil (unparam)
  func copyTempFile(t *testing.T, src, destName string) (string, error) {
```

**Pull request merged**